### PR TITLE
First part 

### PR DESCRIPTION
**Pull request by Baldinini**

### CODE REVIEW

**Review by IgorBochkor — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what is missing?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            if (command.equalsIgnoreCase("stop")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: what about ArrayIndexOutOfBoundException? let’s catch it here as well

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                System.out.println("Please, enter correct data");
+            }
+            betDao.add(bet);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let’s save  the bet if it is not null

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]:
```suggestion
            System.out.println(bet);
```

File: src/main/java/core/basesyntax/dao/BetDaoImpl.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Bet;
+import java.util.List;
+
+public class BetDaoImpl implements BetDao {
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/dao/BetDaoImpl.java]: redundant empty line

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private static final BetDao betDao = new BetDaoImpl();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's not make it static

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**